### DISCUSSION TITLE
differentially acessible peaks between clusters for scATAC

### DISCUSSION THREAD

**Discussion by pengxin2019** [stuart-lab/signac#654] [Q&A] [answered]
Hi Tim,
In terms of the VlnPlot plot created in the Find differentially accessible peaks between clusters here: https://satijalab.org/signac/articles/pbmc_vignette.html

Screen Shot 2021-06-07 at 7 36 44 AM
what is the creteria to define that this gene is differentially expressed in these two clusters? I can see that the p value is extremely small which is reasonalble to say they are different. But are you using the average expresssion value in both groups? It seems like (from the plot) their average are kind of close.

Also, I am wondering what is the meaning of the vertical bar in the CD14 Mono group in this plot. why we have this bar in this group?

In addition, I see that you are using the "FindMarkers" function to find markers between two groups. Do you think it performs better than FindAllMarker which can give us the markers for each cell type compared to all other cell types?

Thanks

**Comment by timoast** [comment, marked as answer]
> what is the creteria to define that this gene is differentially expressed in these two clusters?

Please see the documentation for `Seurat::FindMarkers()`

We compute the log2 fold-change mean accessibility between the groups.

> Also, I am wondering what is the meaning of the vertical bar in the CD14 Mono group in this plot. why we have this bar in this group?

This vertical bar is the density distribution of the violin plot. Since the density is very low for higher values, it appears to be a flat line.

> In addition, I see that you are using the "FindMarkers" function to find markers between two groups. Do you think it performs better than FindAllMarker which can give us the markers for each cell type compared to all other cell types?

The `FindMarkers()` function allows you to compare individual groups of cells, whereas the `FindAllMarkers()` performs a one-vs-all comparison for each group of cells. Which is better will depend on your specific question or goal.
